app/src/main/java/com/codesoom/assignment/controllers/HelloController.java

```diff
@@ -5,6 +5,12 @@
 
 @RestController
 public class HelloController {
+
+    /**
+     * 서버 생존 확인용 문구를 응답합니다
+     *
+     * @return String
+     */
```

Comment:
이건 `@return`이 없어도 괜찮아 보입니다.
```suggestion
    /**
     * 서버 생존 확인용 문구를 응답합니다
     */
```